<?xml version="1.0"?>
<launch>
   <node name="rover_dead_reckoning_node" pkg="rover_deadreckoning" type="rover_dead_reckoning_node" output="screen">
     <rosparam
      command="load"
      file="$(find rover_deadreckoning)/config/config.yaml"
      subst_value="true"
     />
   </node>
</launch>
